lib/mri/neo4j/driver/bolt/wire.rb

```
@@ -61,14 +71,14 @@ def enqueue(message, handler)
             offset += size
           end
           @outbound << END_MARKER
-          @handlers.push(handler)
+          @mutex.synchronize { @handlers.push(handler) }
           self
         end
 
         def pending_outbound? = !@outbound.empty?
```

RuboCop (1.88.0)

[fatal] unexpected token tEQL
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)